src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -106,6 +119,10 @@ if (__DEV__) {
     } else if (badStyleValueWithSemicolonPattern.test(value)) {
       warnStyleValueWithSemicolon(name, value);
     }
+
+    if (Number.isNaN(value)) {
```

Comment:
Jim, I've added a module that exports that function rather than polyfilling onto the Number object. I found a file that appeared to be doing this for Object.assign, but I couldn't tell if that was being used anymore since there is also a node_module object-assign, but tried to follow this convention anyways. Let me know if this is the wrong place for this file, or if adding the isNaN function onto the Number object is preferable, and where that file should be placed. Thanks!